src/components/Property/PropertyLabel.tsx

```diff
@@ -0,0 +1,43 @@
+import { JSX } from 'react';
+import { Text, Tooltip } from '@mantine/core';
+
+import CopyUriButton from '@/components/CopyUriButton/CopyUriButton';
+import { Uri } from '@/types/types';
+
+import { Label } from '../Label/Label';
+
+interface Props {
+  name: string;
+  description: string | JSX.Element;
```

Comment:
```suggestion
  description: JSX.Element;
```

Reply:
Nope: 
![image](https://github.com/user-attachments/assets/573a6634-1869-4525-901d-f948b2088bf8)

But this should be `string | React.JSX.Element;`